### PR TITLE
Switch from pkg/secrets to pkg/subscriptions

### PR DESCRIPTION
**Pull request by rhatdan**
The buildah/pkg/secrts package was move to
containers/common/pkg/subscriptions.
Switch to using this by default.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM

File: docs/source/markdown/podman-build.1.md [hunk 1]
```diff
@@ -844,9 +848,10 @@ $ podman build -f dev/Containerfile https://10.10.10.1/podman/context.tar.gz
 
 ### `.dockerignore`
 
-If the file .dockerignore exists in the context directory, `podman build` reads
-its contents. Podman uses the content to exclude files and directories from
-the context directory, when executing COPY and ADD directives in the
+If the file .dockerignore exists in the context directory, `buildah copy` reads
+its contents. Use `--ignorefile` flag to overide .dockerignore path location.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:852]:
```suggestion
its contents. Use the `--ignorefile` flag to override .dockerignore path location.
```

**Comment by TomSweeneyRedHat** [comment]
Couple small nits and an unhappy test.  Otherwise LGTM

**Comment by vrothberg** [comment]
/hold

Let's wait for 2.2 to be released before merging

**Comment by rhatdan** [comment]
@vrothberg Anything with the 3.0 label should NOT be merged until 2.* is done.

**Pull request merged**